tests/datasets/mock_datasets/population/neher_labs_translated.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Might want to write a more mechanistic test rather than a hard-coded test, this might be hard to maintain if formats change

Reply:
This is the format we want to change things to, so this one should remain the same unless we actually want to change the format our algorithm is using.